src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -0,0 +1,490 @@
+<template>
+  <div>
+
+    <confirm-dialog
+      ref="confirm"
+      attach="#app"
+    />
+
+    <v-expand-transition id="addShareStructureContainer">
+      <ul class="list add-share-structure">
+        <li class="add-share-structure-container">
+          <div class="meta-container">
+            <label class="add-share-structure-header">
+              <span v-if="activeIndex === -1">Add Share {{ shareStructure.type }}</span>
+              <span v-else>Edit Share {{ shareStructure.type }}</span>
+            </label>
+
+            <div class="meta-container__inner">
+              <v-form
+                ref="shareStructureForm"
+                class="share-structure-form"
+                v-model="formValid"
+                v-on:submit.prevent="addShareStructure">
+                <v-text-field
+                  filled
+                  :label="shareStructure.type + ' Name [Shares]'"
+                  :hint="'Enter the name of the  '+ shareStructure.type.toLowerCase() +
+                  '  - the words &quot;Shares&quot; is automatically added'"
+                  id="txt-name"
+                  v-model="shareStructure.name"
+                  :rules="getNameRule()"
+                  suffix="Shares"
+                  persistent-hint/>
+
+                <v-divider class="separator" />
```

Comment:
In other places, separators have a left and right margin (which I think looks better). You may want to add class "mx-4" here and elsewhere.

Reply:
Sure! Updated in next commit